modules/admin-ui-picard/app/src/thunks/eventDetailsThunks.js

```diff
@@ -545,6 +555,219 @@ export const deleteCommentReply = (eventId, commentId, replyId) => async () => {
 }
 
 
+// thunks for scheduling
+
+export const fetchSchedulingInfo = (eventId) => async (dispatch) => {
+    try {
+        dispatch(loadEventSchedulingInProgress())
+
+        // get data from API about event scheduling
+        const schedulingRequest = await axios.get(`/admin-ng/event/${eventId}/scheduling.json`);
+        const schedulingResponse = await schedulingRequest.data;
+
+        // get data from API about capture agents
+        const captureAgentsRequest = await axios.get(`/admin-ng/capture-agents/agents.json`);
+        const captureAgentsResponse = await captureAgentsRequest.data;
+
+        const startDate = new Date(schedulingResponse.start);
+        const endDate = new Date(schedulingResponse.end);
+        const duration = (endDate - startDate) / 1000;
+        const durationHours = (duration - (duration % 3600)) / 3600;
+        const durationMinutes = (duration % 3600) / 60;
```

Comment:
you implemented a similar thing in another file, maybe write a function calculating the duration in `dateUtils.js`